related-tooling.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Related Tooling
+sidebar: home_sidebar
+permalink: /related-tooling/
+toc: false
+---
+
+Open Source tooling, such as for managing the state of Pulp and integrating it with other projects, that is either maintained by the commnunity or other projects, or that is developed by the Pulp Project but not yet in production.
+
+## Foreman / Katello
+
+[Foreman](https://theforeman.org/) is a complete lifecycle management tool for physical and virtual servers. [Katello](https://theforeman.org/plugins/katello/) brings the full power of content management alongside the provisioning and configuration capabilities of Foreman.
+
+Foreman & Katello use Pulp for managing repositories software packages. It provides a graphical WebUI for Pulp and integrates it with its workflows, but does not expose every underlying feature & content type of Pulp.
+
+Foreman & Katello are currently in the incremental [process of migrating from Pulp 2 to Pulp 3.](https://community.theforeman.org/t/update-on-katello-pulp3-integration-and-upcoming-roadmap/15899)
+
+## Ansible Modules for Pulp aka Squeezer
+
+[This project](https://github.com/mdellweg/ansible_modules_pulp) provides ansible modules (currently only for pulp-file) to control a pulp3 server in a descriptive way. This is neither to be confused with ansible-pulp to install pulp, nor pulp_ansible to manage ansible content in pulp. It has been wildly inspired by the sister project [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules).
```

Comment:
"wildly inspired"? maybe widely or largely?

Reply:
Actually, i think this last sentence does not carry any useful information for the user either.
It is part of the repo description, because i was a bit lazy to write better documentation for developers.

Reply:
As a user, I would find the fact that it complements another project very useful. But I will defer to the author's judgement ;)

Reply:
Most of the original text is a placeholder anyway. Maybe it is about time to rethink that readme file.
Given that nothing here is carved in stone, i am fine with either.